src/components/PlayingCard/index.tsx

```diff
@@ -0,0 +1,50 @@
+import './index.scss';
+
+import React, { HTMLProps } from 'react';
+import { Card } from '../../types';
+
+interface PlayingCardProps extends Partial<Card> {
+    size?: 'medium' | 'large' | 'flexible';
+    selected?: boolean;
+    onClick?: HTMLProps<HTMLDivElement>['onClick'];
+    children?: HTMLProps<HTMLDivElement>['children'];
+}
+
+function Marker(props: Partial<Card>) {
+    const { suit, number } = props;
+    const letter = suit?.charAt(0).toUpperCase();
+    return (
+        <div className={`marker marker--${suit}`}>
```

Comment:
should we add a todo comment here for replacing these with icons?

Reply:
no but we can make a ticket for it though; I don't plan on looking at that until after user testing

Reply:
https://github.com/creisle/quarantined_wizzards/issues/3